Cargo.lock

```diff
@@ -4921,25 +4929,25 @@ checksum = "5d58e362dc7206e9456ddbcdbd53c71ba441020e62104703075a69151e38d85f"
 dependencies = [
  "base64 0.22.1",
  "indexmap 2.2.6",
- "metrics",
+ "metrics 0.22.3",
  "metrics-util",
  "quanta",
  "thiserror",
 ]
 
 [[package]]
 name = "metrics-process"
-version = "2.0.0"
+version = "2.1.0"
```

Comment:
please also bump the metrics* dependencies versions to deduplicate `metrics` itself, we were just missing metrics-process

Reply:
bumped